config/routes.rb

```diff
@@ -1,11 +1,10 @@
 # frozen_string_literal: true
 
 Hotsheet::Engine.routes.draw do
-  root to: "pages#index"
-
-  post :broadcast_edit_intent, to: "pages#broadcast_edit_intent"
-
-  Hotsheet.models.each do |model|
-    resources model.table_name, controller: :pages, only: %i[index update], model: model.name
+  Hotsheet.sheets.each_key do |sheet_name|
+    resources sheet_name, only: %i[index update], controller: :sheets, sheet_name:
```

Comment:
```suggestion
    resources sheet_name, only: %i[index update], controller: :sheets, sheet_name: sheet_name
```
I don't like mixing this up. When reading this, it looks like it's missing for me